Src/gpio.c

```diff
@@ -5,7 +5,7 @@
  *      Author: monicapinheiro
  */
 
-#include "gpio.h"
+#include "../Inc/gpio.h"

```

Comment:
@engmonicapinheiro Does this work with relateive paths?

Or do we need to change something so that we can use `gpio.h` and the place where the compiler searches for it is always `Src`?